### DISCUSSION TITLE
All Photos were indexed - But several are missing (400+)

### DISCUSSION THREAD

**Discussion by saber-rider1** [photoprism/photoprism#1717] [Q&A] [answered]
Hi there,

i have indexed all my photos on my server.
But some are missing.

For example one folder on the server contains 500+ photos.
But photoprism can find only 56.

I deactivate the quality filter and run the indexing of that folder again. But nothing changed.

I´m sure, there is a mistake from my side.
But what?

Do you have a hint for me?

Greets
saber

**Comment by lastzero** [comment, marked as answer]
Here is a checklist for you: https://docs.photoprism.org/getting-started/troubleshooting/#missing-pictures

**Reply by saber-rider1** [reply]
Thank you. I will check.
